MANIFEST.in

```diff
@@ -8,6 +8,7 @@ exclude .dvcignore
 exclude .gitignore
 exclude .pre-commit-config.yaml
 exclude .readthedocs.yaml
+exclude .mailmap
```

Comment:
It seems setuptools_scm automatically exclude the `.mailmap` file from distributions, but I guess it does no harm to explicitly exclude it here.